recipes/pypolyagamma-3/meta.yaml

```diff
@@ -1,43 +1,46 @@
 {% set name = "pypolyagamma-3" %}
-{% set version = "0.1.10" %}
+{% set version = "0.1.11" %}
 
 package:
-  name: {{ name }}
-  version: {{ version }}
+    name: {{ name }}
+    version: {{ version }}
 
 source:
-  url: https://github.com/Savvysherpa/pypolyagamma/archive/{{ version }}.tar.gz
-  fn: {{ name }}-{{ version }}.tar.gz
-  sha256: 6da114c6398081e8c5e5efca958e7c6e163acc7c9d0ef2c7448cbc35c3382b2d
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/Savvysherpa/pypolyagamma/archive/{{ version }}.tar.gz
+    sha256: 6da114c6398081e8c5e5efca958e7c6e163acc7c9d0ef2c7448cbc35c3382b2d
```

Comment:
...or you can enter the wrong checksum in the recipe, let `conda` tell you the mismatch, and update accordingly. In any event, it does feel funky as @ocefpaf has said.